notebooks/CubevizExample.ipynb

```diff
@@ -61,11 +68,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "fn = download_file('https://stsci.box.com/shared/static/28a88k1qfipo4yxc4p4d40v4axtlal8y.fits', cache=True)\n",
+    "data_dir = tempfile.gettempdir()\n",
+    "#data_dir = \"/User/username/Data/\"\n",
```

Comment:
Still not a big fan but it's a good compromise until we have the caching debate later. Thanks!

Reply:
It would be nice to cache by default, but at least we can get this in now and have the discussion on the HOW to cache not block this PR

Reply:
I think right now, by default it won't redownload if it detects the file is already there.

Reply:
`cache=True` by default in `download_file`.

Reply:
Ahh, right. Thanks!

Reply:
Using `cache=True` for any of the astroquery downloading methods can be a bit dangerous with MAST JWST products, especially if you're getting _uncal.fits products (and some others), as updated calibrations of the files can have the exact same size as the old versions, and if you do this generally, you'll get a mixed lot of old and new files.  This is a feature (bug?) of the FITS format.  So something to be aware of.

Reply:
Hey, stranger!

Shouldn't cache invalidation be MAST's problem and not ours? 😉

Reply:
Would adding a note saying to add `cache=False` to force a download be helpful here?

Reply:
I did a double take when I saw @jdavies-st in my morning email digest! Good to see ya :)

I don't think this is a major concern for these example notebooks since these notebooks are just... well examples. I'd hope people aren't taking these notebooks and doing real science on them!

Reply:
> hope people aren't taking these notebooks and doing real science on them

😬 🙊

Reply:
I like the idea of a note in the notebook, though, in case someone copies the logic for their own data from MAST and might not understand this... feature.

Reply:
> Shouldn't cache invalidation be MAST's problem and not ours? 😉

This is actually a problem of `astroquery`, not specific to MAST.  Basically to see if the file it wants to download is cached, it checks the size.  It does not do a checksum or hash.  But with FITS, two different files can have the same size, and we've actually seen precisely this for JWST _uncal.fits data (and some other data products), which have quite different calibrations.  All FITS header cards are the same size, even if the content of the keyword is different.

Of course there's big overhead to doing checksums, and that's probably a good enough reason as any for astroquery to not being doing it routinely.  But just something to be aware of.  =)